release/models/bgp/openconfig-bgp-common.yang

```diff
@@ -696,21 +702,57 @@ submodule openconfig-bgp-common {
 
   grouping bgp-common-use-multiple-paths-link-bandwidth-config {
     description
-      "Parameters controlling usage of  of DMZ Link-Bandwidth
-      extended community in pultipath RIB/FIB formation";
+      "Parameters controlling usage of DMZ Link-Bandwidth
+      extended community in multipath RIB/FIB formation";
+
     leaf enabled {
       type boolean;
       description
-      "When set to TRUE, BGP multiplepath shall distributed traffic
-      load among contributing routes proportionally to value of
-      Local Administrator subfield of link-bandwidth extended
-      community [draft-ietf-idr-link-bandwidth-07].
-      This leaf has no effect if BGP multi-path is disabled or
-      if maximum-path attribute of BGP multi-path value is set
-      to 1";
+        "When set to TRUE, BGP multiplepath shall distributed traffic
+        load among contributing routes proportionally to value of
+        Local Administrator subfield of link-bandwidth extended
+        community [draft-ietf-idr-link-bandwidth-07].
+        This leaf has no effect if BGP multi-path is disabled or
+        if maximum-path attribute of BGP multi-path value is set
+        to 1";
```

Comment:
This is a bit of a head-scratching sentence -- perhaps it should say:

"When set to true, and BGP is configured to use multiple paths, traffic is distributed using the proportions set by the Local Administrator subfield of the Link Bandwidth extended community [draft...]. This leaf has no effect if BGP multi-path is disabled, or if the maximum number of paths is set to 1."